provider/ansible/example.rb

```diff
@@ -27,8 +27,8 @@ module Ansible
     }.freeze
 
     EXAMPLE_DEFAULTS = {
-      name: 'testObject',
-      project: 'testProject',
+      name: 'test_object',
+      project: 'test_project',
```

Comment:
What's this doing in here?

Reply:
Should've changed the PR message on this.

Turns out that 'testObject' isn't a valid object name. Wanted to change this and this PR seemed like a half way decent place to do that.